tee-worker/service/src/config.rs

```diff
@@ -186,7 +202,8 @@ impl From<&ArgMatches<'_>> for Config {
 			run_config,
 			m.value_of("running-mode").unwrap_or(DEFAULT_RUNNING_MODE).to_string(),
 			is_mock_server_enabled,
-			u16::from_str(mock_server_port).unwrap(),
+			mock_server_port.to_string(),
```

Comment:
How mock_server_port not added before? 😓

Reply:
It was added, but as type `u16`